changelog/dep_d1_ops.dd

```diff
@@ -0,0 +1,68 @@
+D1 operator overloads have been deprecated.
+
+The following D1 operator overloads have been deprecated in favor of `opUnary`:
+  * `opNeg`
+  * `opCom`
+  * `opPostInc`
+  * `opPostDec`
+  * `opStar`
+
+The following D1 operator overloads have been deprecated in favor of `opBinary`:
+  * `opAdd`
+  * `opSub`
+  * `opMul`
+  * `opDiv`
+  * `opMod`
+  * `opAnd`
+  * `opXor`
+  * `opOr`
+  * `opShl`
+  * `opShr`
+  * `opUShr`
+  * `opCat`
+  * `opIn`
+
+The following D1 operator overloads have been deprecated in favor of `opBinaryRight`:
+  * `opAdd_r`
+  * `opSub_r`
+  * `opMul_r`
+  * `opDiv_r`
+  * `opMod_r`
+  * `opAnd_r`
+  * `opXor_r`
+  * `opOr_r`
+  * `opShl_r`
+  * `opShr_r`
+  * `opUShr_r`
+  * `opCat_r`
+  * `opIn_r`
+
+The following D1 operator overloads have been deprecated in favor of `opOpAssign`:
+  * `opAddAssign`
+  * `opSubAssign`
+  * `opMulAssign`
+  * `opDivAssign`
+  * `opModAssign`
+  * `opAndAssign`
+  * `opOrAssign`
+  * `opXorAssign`
+  * `opShlAssign`
+  * `opShrAssign`
+  * `opUShrAssign`
+  * `opCatAssign`
+
+Starting with this release, any D code that triggers a lowering to the D1 operator overloads will emit a deprecation warning.
+
+---
+struct S
+{
+    int opAdd(int i) { ... }
+}
+
+void main()
+{
+    S s;
+    int i;
+    i = s + 1;  // Deprecation: `opAdd` is deprecated.  Use `opBinary` instead.
+}
+---
```

Comment:
While it's important to show what's wrong, it's also quite important to show how to fix things.
It would be great if this example (or an additional one) showed how the code can be fixed to be compliant with the new rules.